### ISSUE TITLE
Question about how to install inplace-abn

### ISSUE THREAD

**Issue by FFFFlamingo** [fcdl94/MiB#79]
Thanks for your great work!
How do I install the corresponding version of Inplace-ABN on  Windows?
'Pip install inplace-abn' doesn't work.Neither does 'git clone' the package and then  'python setup.py install'.
I look forward to your reply.

**Comment by fcdl94** [comment]
Hello. Please refer to this repository for the install instruction: https://github.com/mapillary/inplace_abn

**Comment by cs-mshah** [comment]
I tried to follow their instructions but they throw errors.
https://pastebin.com/BqUnhDSW

**Comment by fcdl94** [comment]
It seems you have trouble compiling their library. Can you please check if the CUDA and GCC you are using match the requirement? 
Did you try to install the library from pip?

**Comment by cs-mshah** [comment]
Yes all requirements are satisfied. "NOTE 2: we are only able to provide support for Linux platforms and CUDA versions >= 10.0" I am using CUDA 11.7.  
I tried using both pip and from source as given in their instructions. Both give the same errors.  
Can this repository code be updated to account for new changes: using `torch.amp` and `from torch.nn.parallel import DistributedDataParallel as DDP`

**Comment by fcdl94** [comment]
Take a look at Mibv2 here: https://github.com/fcdl94/MiBv2/

**Comment by cs-mshah** [comment]
Thanks for the new implementation. I tried to create an environment using `requirements.txt` but the inplace-abn package doesn't correctly get installed and I believe the model uses it. Is there a way to do without inplace-abn or can you push the inplace-abn version which works to the repo?

**Comment by fcdl94** [comment]
Inplace-abn installation is not under my control 😅  Please if you are not able to install it, check their repository. In-place ABN is mandatory for the project, there is no specific version for it (I think only one was released).

**Comment by prinshul** [comment]
Hi @fcdl94 

what is the CUDA version you used for your code?

**Comment by fcdl94** [comment]
It was CUDA 10.2.
